validation.js

```diff
@@ -0,0 +1,106 @@
+const email = document.getElementById("email");
+const age = document.getElementById("age");
+const password = document.getElementById("password");
+const confirmPassword = document.getElementById("confirm-password");
+const btn = document.getElementById("signup-btn");
+const contact = document.getElementById("contact");
+
+
+var password1 = "";
+var password2 = "";
+var passPattern = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?!.*\s).{8,15}$/;
+var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+var validPassowrd = false;
+var validMatchPassword = false;
+var validContact = false;
+var validEmails = false;
+var validAge = false;
+
+password.addEventListener("change", validatePassword);
+confirmPassword.addEventListener("change", matchpassword);
+
+function validatePassword() {
+    console.log(password.value);
+    password1 = password.value;
+    if (password1.match(passPattern) == null) {
+        setTextClass(password, "txt-password", "your passowrd should be atleast 8 digits long and should have atleast one aplhabet one number one upper case and one lowercase", "add")
+        document.getElementById("txt-password").innerHTML = "your passowrd should be atleast 8 digits long and should have atleast one aplhabet one number one upper case and one lowercase";
+    } else {
+        validPassowrd = true;
+        document.getElementById("txt-password").innerHTML = "";
+        setTextClass(password, "txt-password", "", "remove")
+    }
+}
+
+function matchpassword() {
+    password2 = confirmPassword.value;
+    if (password1!= password2) {
+        setTextClass(confirmPassword, "txt-matchpassword", "passowrd not matched", "add");
+    } else {
+        document.getElementById("txt-matchpassword").classList.remove("text-danger");
+        document.getElementById("txt-matchpassword").classList.add("text-success");
+        setTextClass(confirmPassword, "txt-matchpassword", "passowrdmatched", "remove");
+        validMatchPassword = true;
+    }
+}
+
+contact.addEventListener("change", function () {
+
+    if (contact.value.length == 11) {
```

Comment:
also use regrex here to check for that all the letters are number or other character uses for number like +